blockly-rtc/src/http_handlers.js

```diff
@@ -20,10 +20,18 @@
  * @author navil@google.com (Navil Perez)
  */
 
+/**
```

Comment:
Is there a single place that you can put all of these typedefs, so that you don't need to have them repeated in multiple files?

Reply:
Not having them repeated would be ideal. Based on a quick Google search it seems that putting them all in a single `config.js` file can be done.

Is there a convention for this that you recommend?

Reply:
That sounds good.  I would call it `typedefs.js` instead of `config.js`.  You can add that to your task list and do it in a separate PR.